etc/settings.json

```diff
@@ -16,6 +16,9 @@
   "RATE_LIMIT_PATCH": null,   /* Limit number of PATCH requests */
   "RATE_LIMIT_DELETE": null,  /* Limit number of DELETE requests */
 
+  /* Default date format
+  "DATE_FORMAT": '%a, %d %b %Y %H:%M:%S GMT', */
+
```

Comment:
Not sure have possibility change the date is a good idea, because I use it (_created field) in module backend in arbiter (not sure at 100%).